playbooks/roles/aws/tasks/main.yml

```diff
@@ -124,5 +124,5 @@
     mode: "0644"
   with_items: "{{ motd_files_exist.results }}"
   when:
-    - vagrant_home_dir.stat.exists == False and ansible_distribution in common_debian_variants and item.stat.exists
     - cloud_provider == "aws"
+    - vagrant_home_dir.stat.exists == False and ansible_distribution in common_debian_variants and item.stat.exists
```

Comment:
or completely remove it
```suggestion
```

Not sure which is the right one. But sure there shouldn't be any Vagrant code anymore.